Tests/ExampleExtensions/ExampleExtensions.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>net47;net6.0</TargetFrameworks>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>disable</Nullable>
+        <IsPackable>false</IsPackable>
+        <LangVersion>11</LangVersion>
```

Comment:
Inherit this from https://github.com/fluentassertions/fluentassertions/blob/develop/Directory.Build.props
```suggestion
```